src/nandini/Exam1.java

```diff
@@ -0,0 +1,51 @@
+package nandini;
+
+public class Exam1 {
+
+	 String getMaxLengthName(String[] arr){
+		int i =0; 
+		int length = 0;
+		int maxLengthName = arr[0].length();
+		for(int index = 1; index < arr.length; index++) {
+			
+			if(arr[index].length() > maxLengthName) {
+				i = index;
+				maxLengthName = arr[index].length();
+			}
+			    
+		}
+
+		length = arr[i].length();
```

Comment:
separate length variable is not really required. we can directly use arr[i].length() in sop